Create how to resize virtual clusters (#1111)
spectrocloud/librarium: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md

---
title: "Resize Palette Virtual Clusters"
metaTitle: "Resize Palette Virtual Clusters"
metaDescription: "Learn how to resize Palette Virtual Clusters"
hideToC: false
fullWidth: false
---

import Tabs from 'shared/components/ui/Tabs';
import InfoBox from 'shared/components/InfoBox';
import WarningBox from 'shared/components/WarningBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';
import Tooltip from "shared/components/ui/Tooltip";

# Overview

> You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify can be greater than the Cluster Group allocation. 

Reviewer: ```suggestion
You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify cannot be greater than the system-level quota and the tenant user quota. To learn more about resource quotas, refer to the [resource quota](/devx/resource-quota) documentation.
```

The resource quota page has been updated with rich diagrams and examples. That should help address any confusion.

Author: done.

Author: added suggestion.

# Prerequisite

* A running [Palette Virtual Cluster](/devx/palette-virtual-clusters/pause-restore-virtual-clusters).

# Resize Virtual Clusters

Use the following steps to resize a virtual cluster.

> 1. Log in to [Palette](https://console.spectrocloud.com).

Reviewer: Rita, for these bullets, let's add another newline in between them so it looks less compressed when displayed.

Author: They each had a new line between them, but I added a break to space them out.

2. In App Mode, click **Virtual Clusters** in the **Main Menu**.

3. Select the virtual cluster you want to resize, and click **Settings > Cluster Settings**.

> 4. Click **Cluster Size** and specify new resource allocations for your virtual cluster. The size you specify can be greater than the Cluster Group allocation.

Reviewer 2: The size you specify cannot be greater than the Cluster Group limits.

Reviewer: I came here to say the same. @tiwarisumit3 I think I have an idea of how to handle this. Adding a suggestion about.

Author: fixed.

> 4. Click **Cluster Size** and specify new resource allocations for your virtual cluster. The size you specify can be greater than the Cluster Group allocation.

Reviewer: ```suggestion
4. Click **Cluster Size** and specify new resource allocations for your virtual cluster.
```

Author: deleted second sentence.

5. Save your changes.

# Validation

> To verify your changes, click **Virtual Clusters** in the **Main Menu** and select the resized cluster. The virtual cluster Overview page displays the new **Allocated Quota** for the cluster.

Reviewer: ```suggestion
To verify your changes, click **Virtual Clusters** in the left **Main Menu** and select the resized cluster. The virtual cluster Overview page displays the new **Allocated Quota** for the cluster.
```

Author: added left.

Author: added left

---
title: "Resize Palette Virtual Clusters"
metaTitle: "Resize Palette Virtual Clusters"
metaDescription: "Learn how to resize Palette Virtual Clusters"
hideToC: false
fullWidth: false
---

import Tabs from 'shared/components/ui/Tabs';
import InfoBox from 'shared/components/InfoBox';
import WarningBox from 'shared/components/WarningBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';
import Tooltip from "shared/components/ui/Tooltip";

# Overview

You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify cannot be greater than the system-level quota and the tenant user quota. 

# Prerequisite

* A running [Palette Virtual Cluster](/devx/palette-virtual-clusters/pause-restore-virtual-clusters).

# Resize Virtual Clusters

Use the following steps to resize a virtual cluster.
<br />

1. Log in to [Palette](https://console.spectrocloud.com).
<br />

2. In App Mode, click **Virtual Clusters** in the **Main Menu**.
<br />

3. Select the virtual cluster you want to resize, and click **Settings > Cluster Settings**.
<br />

> 4. Click **Cluster Size** and specify new resource allocations for your virtual cluster. The size you specify cannot be greater than the system-level quota and the tenant user quota. To learn more about resource quotas, refer to the [resource quota](/devx/resource-quota) documentation.

Reviewer 2: The size you specify cannot be greater than the system-level quota for system level cluster group like Beehive and the tenant user quota for tenant level cluster groups

Author: Added.

> 4. Click **Cluster Size** and specify new resource allocations for your virtual cluster. The size you specify cannot be greater than the system-level quota for a cluster group like Beehive and the user quota for tenant-level cluster groups. To learn more about resource quotas, refer to the [resource quota](/devx/resource-quota) documentation.

Reviewer 2: The size you specify cannot be greater than the system-level quota for a cluster group like Beehive or the user quota for tenant-level cluster groups.

Author: changed and to or.

Author: changed and to or
